conans/client/conf/__init__.py

```diff
@@ -497,6 +504,14 @@ def full_transitive_package_id(self):
         except ConanException:
             return None
 
+    @property
+    def msvc_visual_incompatible(self):
+        try:
+            visual_comp = self.get_item("general.msvc_visual_incompatible")
+            return visual_comp.lower() in ("1", "true")
+        except ConanException:
+            return None
```

Comment:
Here I prefer to return `False` as it is an explicit default value. Returning `None` is like letting the consumer choose any value. Does it make sense?